tests/test.js

```diff
@@ -80,15 +80,15 @@ test.beforeEach(async (t) => {
   t.context.stub = sinon.stub(AWS, 'Lambda')
     .returns({
       getLayerVersionByArn: () => ({
-        promise: async () => ({
+        promise: async() => ({
```

Comment:
Interesting. I thought that a space before the argument parentheses was required, but I guess not if this passes lint

Reply:
Apparently not by default lint rules.  I didn't want to *update* those as part of this PR, but we should probably attempt to make them consistent with the rest of cumulus